ingenialink/poller.py

```diff
@@ -38,6 +38,7 @@ def cancel(self):
         if self.thread.is_alive():
             self.thread.join()
 
+lock_poller = Lock()
```

Comment:
The poller should use the same lock as the active communication. If you create two different locks, the exclusion mechanism is not going to work.

Reply:
As @MartinALL91 comments [here](https://github.com/ingeniamc/ingenialink-python/pull/309/files#r1291041168), the lock is not needed for the reading. It should only affects that the sample that a channel is reading will late milliseconds more. In fact, threads are using the same lock finally, which are the locks of each servo.

However, I would keep this lock because of [this](https://github.com/ingeniamc/ingenialink-python/pull/309/files#diff-828b231da05ac56f01ef705bb7514038b48b067402fba9c7d4fd4e409ac4577aR246). In this case, as it's another shared resource, I think it makes sense we use another lock.

What do you think?